### PR TITLE
c1-mauer-rose

### PR DESCRIPTION
**Pull request by kfahn22**
Adding c1-mauer-rose to coding challenges.  I have the rest  of coding in the cabana as well, but will create pull requests in order.

Note:  I do not have the index.png.

### CODE REVIEW

File: content/videos/challenges/c1-mauer-rose/index.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
+{
+    "title": "Maurer Rose",
+    "description": "I code a visualization of the Maurer Rose pattern.",
```
Comment (shiffman) [inline content/videos/challenges/c1-mauer-rose/index.json:3]:
```suggestion
    "description": "Coding in the Cabana is a series where I attempt challenges from my garden cabana in Brooklyn, NY. In this episode, I code a visualization of the Maurer Rose patter in JavaScript (p5.js).",
```

File: content/videos/challenges/c1-mauer-rose/index.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    "videoNumber": "c1",
```
Comment (shiffman) [inline content/videos/challenges/c1-mauer-rose/index.json:4]:
```suggestion
    "videoNumber": "1 - Coding in the Cabana",
```

File: content/videos/challenges/c1-mauer-rose/index.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      { "time": "0:00", "title": "Maurer Rose" },
+      { "time": "3:32", "title": "Coding Starts" },
+      { "time": "5:44", "title": "Maurer Rose Algorithm" },
+      { "time": "8:30", "title": " Visualizing With a Rose Pattern" },
+      { "time": "10:05", "title": "Adding a Slider" },
+      { "time": "10:52 ", "title": "New ideas And Goodbyes" }
+    ],
+    "codeExamples": [
+      {
+        "title": "Mauer Rose",
```
Comment (shiffman) [inline content/videos/challenges/c1-mauer-rose/index.json:20]:
```suggestion
        "title": "Maurer Rose",
```

File: content/videos/challenges/c1-mauer-rose/index.json [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        "description": "Implementation of mauer rose",
```
Comment (shiffman) [inline content/videos/challenges/c1-mauer-rose/index.json:21]:
```suggestion
        "description": "Visualization of the Maurer rose pattern",
```

File: content/videos/challenges/c1-mauer-rose/index.json [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+        "image": "rose.png",
+        "urls": {
+          "p5": "https://editor.p5js.org/codingtrain/sketches/qa7RiptE9",
+          "processing": "https://github.com/CodingTrain/website-archive/tree/main/challenges/coding-in-the-cabana/001_maurer_rose/Processing"
```
Comment (shiffman) [inline content/videos/challenges/c1-mauer-rose/index.json:25]:
```suggestion
          "processing": "https://github.com/CodingTrain/Coding-in-the-Cabana/tree/main/001-Maurer-Rose/"
```

**Review by shiffman — COMMENTED** [review]
Thank you for this! Made some minor adjustments, note the addition of a new Repo for Coding in the Cabana example code!

https://github.com/CodingTrain/Coding-in-the-Cabana/

**Comment by kfahn22** [comment]
I have edited the remaining challenges in Coding in the Cabana.  (I wasn't sure what was changed in the third commit, though.)  Here are the edits I made.
-updated the video number
-updated the description of the video
-updated the description for example code
-updated the link to repo
Please let me know if there is anything else to change.

**Comment by shiffman** [comment]
All sounds good! I have to move the other code examples over too! Let me merge this one and we can tackle the others in separate pull requests!

**Pull request merged**